website/database.py

```diff
@@ -314,6 +315,23 @@ def delete_class(self, Class):
     def resolve_class_link(self, link_id):
         return CLASSES.get({'link': link_id})
 
+    def get_username_invite(self, username):
+        return INVITATIONS.get({'username': username})
+
+    def add_class_invite(self, username, class_id):
+        INVITATIONS.create({'username': username, 'class_id': class_id, 'timestamp': timems ()})
+
+    def remove_class_invite(self, username):
+        INVITATIONS.delete({'username': username})
+
+    def get_class_invites(self, class_id):
+        invites = INVITATIONS.scan()
```

Comment:
It would! Currently we only use "username" as partition key to retrieve the user invites. I thought this wouldn't be too expensive due to the small amount of classes and the small amount of retrievals. But still, agree that it would be best to add an additional index.

Reply:
Fixed! Would be great if you could take a look.